system-tests/test/results_spec.ts

```diff
@@ -0,0 +1,72 @@
+import path from 'path'
+import { fs } from '@packages/server/lib/util/fs'
+import systemTests from '../lib/system-tests'
+import Fixtures from '../lib/fixtures'
+
+// source: https://www.myintervals.com/blog/2009/05/20/iso-8601-date-validation-that-doesnt-suck/
+const isoDateRegex = /"([\+-]?\d{4}(?!\d{2}\b))((-?)((0[1-9]|1[0-2])(\3([12]\d|0[1-9]|3[01]))?|W([0-4]\d|5[0-2])(-?[1-7])?|(00[1-9]|0[1-9]\d|[12]\d{2}|3([0-5]\d|6[1-6])))([T\s]((([01]\d|2[0-3])((:?)[0-5]\d)?|24\:?00)([\.,]\d+(?!:))?)?(\17[0-5]\d([\.,]\d+)?)?([zZ]|([\+-])([01]\d|2[0-3]):?([0-5]\d)?)?)?)?"/g
+const numberRegex = /"(duration|totalDuration|port)": \d+/g
+const osNameRegex = /"(osName|platform)": "[^"]+"/g
+const archRegex = /"arch": "[^"]+"/g
+const versionRegex = /"(browserVersion|cypressVersion|osVersion|resolvedNodeVersion|version)": "[^"]+"/g
+const majorVersionRegex = /"(majorVersion)": [0-9]+/g
+const pathRegex = /"(absolute|projectRoot|downloadsFolder|fileServerFolder|fixturesFolder|resolvedNodePath|screenshotsFolder|videosFolder|cypressBinaryRoot|path)": "[^"]+"/g
+const stackLineRegex = /"displayError": (.*)at .*/g
```

Comment:
Which is the best test to look at to see if these are doing what they are supposed to? I looked at the snapshots and don't see anything off. Would that be enough to be confident these are correct?

Reply:
I'd say so. The snapshot would fail either on a subsequent run where the durations are concerned and on a different system, like CI, where paths and OS/browser details are concerned.